03_launch_mgmt_cluster.sh

```diff
@@ -78,8 +78,9 @@ EOF
         echo "DEPLOY_ISO_URL=${DEPLOY_ISO_URL}" | sudo tee -a "${BMOPATH}/config/default/ironic.env"
     fi
 
-    # Deploy BMO using deploy.sh script
-    "${BMOPATH}/tools/deploy.sh" -b "${BMO_IRONIC_ARGS[@]}"
+    # Deploy BMO using deploy-cli
+    export BMOPATH="${BMOPATH}"
```

Comment:
```suggestion
    export BMOPATH
```
No need to redefine value